modules/rosa-sdpolicy-account-management.adoc

```diff
@@ -96,6 +105,28 @@ For additional information, see the link:https://kubernetes.io/docs/tasks/admini
 - C5.24xlarge (96 vCPU, 192 GiB)
 ====
 
+.Storage-optimized compute types
+[%collapsible]
+====
+- i3.large (2 vCPU, 15.25 GiB)
+- i3.xlarge	(4 vCPU, 30.5 GiB)
```

Comment:
I double checked the spacing here and in the high frequency section. It is correct, but rendering strangely on the diff view.